lib/puppet/type/user.rb

```diff
@@ -614,8 +622,8 @@ def membership
       desc "The name of the I&A module to use to manage this user."
     end
 
-    newproperty(:attributes, :parent => Puppet::Property::KeyValue, :required_features => :manages_aix_lam) do
-      desc "Specify AIX attributes for the user in an array of attribute = value pairs."
+    newproperty(:attributes, :parent => Puppet::Property::KeyValue, :required_features => :manages_attributes) do
+      desc "Specify additional attributes for the user in a hash (or array) of attribute = value pairs."
```

Comment:
Should I describe the available Windows attributes here? Would it be better to specify this in the provider instead since that's where it's implemented?

Reply:
Yes, that makes sense. I'll see what other reviewers think then update accordingly with documentation.